Phishing Campaign - GetCampaignIncidentsInfo handling deleted incidents (#27621)
demisto/content: Packs/Campaign/ReleaseNotes/3_3_1.md

#### Scripts

##### GetCampaignIncidentsInfo

> - Fixed an issue causing the script to fail when incidents were deleted.

Reviewer: ```suggestion
- Fixed an issue causing the script to fail after incidents are deleted.
```

Reviewer: @ShirleyDenkberg Which of the options do you think is clearer?
